### DISCUSSION TITLE
can connect to Wi-Fi but not to YI App

### DISCUSSION THREAD

**Discussion by chl3b** [roleoroleo/yi-hack-MStar#549] [General]
Hi Hope this question was not asked before. For some reason app doesnt want synchronise with camera BFUS it is connected to Wi-Fi I can see view in onvifier and VIA VLC but IT doesnt wants to match with cloud, in settings page is enabled yi cloud. Any suggestions? I have also one other dam bricked IFUS cam completely nothing happening just LED is Amber and that is IT. I read some treads about how to unbrick but I am almost numb in this case, thinking to edit fw from the working if this is possible as is BFUS and IFUS or is any dumped backup firmware.
Status.json file displays
{
"name":"yi-hack-mstar",
"hostname":"yi-hack",
"fw_version":"0.4.3",
"home_version":"4.5.0.0C_201910080934",
"model_suffix":"y203c",
"ptz":"no",
"serial_number":"BFUS

**Comment by roleoroleo** [comment]
Never seen this problem.
Try to reset the cam with the button and remove it from the app.
Then repeat the wizard to add the cam in the app.

About the other cam bricked, is it a MStar? If yes, I can send you a couple of file to try the restore procedure.
But I need to know the model or the fw version (I mean Yi firmware).

**Reply by Mjolinor** [reply]
I am having a similar problem. 
I reset the camera.
Went through the yi procedure to make it work just with the app
I installed on the SD card no problem and all worked properly. Still had the yi access and got RSTP, web page etc etc.

I unplugged the camera for a couple of days. Next time I plugged it in the yi ap would not connect, no web page, no RSTP.
I reset the camera and started again but now when I get to the camera scanning the QR code that the phone makes with the wifi information in the QR is scanned correctly and wifi connects but the next step from the app fails. This is with or without the SD card.

Now I have to go through the app procedure and let the camera scan the phone's QR code. At that point I have web access so I have to edit the IP and set it static plus set the wifi details on the web interface. After that the camera remembers the wifi information and all is good apart from the yi app does not have the camera registered (not a problem) but a side effect of this seems to be that the automatic motion detect pan does not work. The camera records to FTP OK when it detects motion but does not pan to follow the movement any more. I can move the camera from the web interface no problem.
